pkg/translate/translatetest/util.go

```diff
@@ -0,0 +1,16 @@
+package translatetest
```

Comment:
why not keep this in pkg/testutil/rand folder?

Reply:
it is specificly tailored for 3rd party translate services integration tests, so i thought it is logicly that is in the same pkg